app/modules/transactions.js

```diff
@@ -67,25 +67,19 @@ export const sendTransaction = (sendAddress: string, sendAmount: string, symbol:
 
     const isHardwareSend = !!publicKey
```

Comment:
Is there a better check for this?

Reply:
Maybe we can define the type of the login? Feels like this condition is only known to us..

Reply:
yeah, that makes sense. I will incorporate those changes for the ledger login page as well